modules/parser/js/src/main/scala/io/circe/parser/package.scala

```diff
@@ -1,17 +0,0 @@
-package io.circe
-
-import io.circe.scalajs.convertJsToJson
-import scala.scalajs.js.JSON
-import scala.util.control.NonFatal
-
-package object parser extends Parser {
-  final def parse(input: String): Either[ParsingFailure, Json] = (
-    try convertJsToJson(JSON.parse(input))
-    catch {
-      case NonFatal(exception) => Left(ParsingFailure(exception.getMessage, exception))
-    }
-  ) match {
-    case r @ Right(_)    => r.asInstanceOf[Either[ParsingFailure, Json]]
-    case Left(exception) => Left(ParsingFailure(exception.getMessage, exception))
-  }
-}
```

Comment:
This may be more complicated, but rather than removing this, would it make more sense to keep it around for some users if they want to use the old `JSON.parse` version?  Certainly switch to jawn by default.

Reply:
Good point. I can move this to the `circe-scalajs` module, wdyt?

Reply:
That make sense to me.